lib/connections/limiter.go

```diff
@@ -97,52 +182,68 @@ func (lim *limiter) String() string {
 	return "connections.limiter"
 }
 
+func (lim *limiter) newReadLimiter(remoteID protocol.DeviceID, r io.Reader, isLAN bool) io.Reader {
```

Comment:
While you're changing this: Wouldn't `newLimitedReader` be a more appropriate name? (same for the writer below)

Reply:
Agree